tony-history-server/conf/tony-site.sample.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<configuration>
+    <property>
+        <name>tony.history.location</name>
+        <value>/system/tony-history</value>
+    </property>
+    <property>
+        <name>tony.keytab.user</name>
+        <value>user1</value>
+    </property>
+    <property>
+        <name>tony.keytab.location</name>
+        <value>/location/to/tony.keytab</value>
+    </property>
+    <property>
+        <name>tony.https.port</name>
+        <value>19886</value>
+    </property>
+    <property>
+        <name>tony.https.keystore.path</name>
+        <value>/location/to/tony.jks</value>
+    </property>
+    <property>
+        <name>tony.https.keystore.type</name>
+        <value>JKS</value>
+    </property>
+    <property>
+        <name>tony.https.keystore.password</name>
+        <value>password</value>
+    </property>
+    <property>
+        <name>tony.https.keystore.algorithm</name>
+        <value>SunX509</value>
+    </property>
+    <property>
+        <name>tony.http.port</name>
+        <value>disabled</value>
+    </property>
+    <property>
+        <name>tony.secret.key</name>
+        <value>Prod</value>
```

Comment:
What does this `secret.key` do? The keystore password makes sense to me, but I'm not sure what this `secret.key` is for. Is the `secret.key` a password of sorts or is it just an identifier/label?

Reply:
https://www.playframework.com/documentation/2.6.x/ApplicationSecret.

Reply:
Thanks. That doc says "Anyone that can get access to the secret will be able to generate any session they please, effectively allowing them to log in to your system as any user they please."

So we should make sure the production secret we use is not checked in to source control and not in the tony-site.xml on the client side.

Reply:
I would suggest changing the value of `tony.secret.key` in the example to `changeme` (which is the default for `play.http.secret.key`.